enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -656,6 +642,13 @@ func (c *ociContainer) doWithStatsTracking(ctx context.Context, invokeRuntimeFn
 }
 
 func (c *ociContainer) setupCgroup(ctx context.Context) error {
+	// Lease CPUs for task execution, and set cleanup function.
+	leaseID := uuid.New()
+	_, leasedCPUs, cleanupFunc := c.env.GetCPULeaser().Acquire(c.milliCPU, leaseID)
```

Comment:
I think we could use the returned node here to set `cpuset.mems`? (maybe in a later PR)

Reply:
yeah, agreed.